spread.yaml

```diff
@@ -253,9 +253,8 @@ prepare: |
         exit 1
     fi
 
-    # apt update is hanging on security.ubuntu.com with IPv6.
-    sysctl -w net.ipv6.conf.all.disable_ipv6=1
-    trap "sysctl -w net.ipv6.conf.all.disable_ipv6=0" EXIT
+    # apt update is hanging on security.ubuntu.com with IPv6, prefer IPv4 over IPv6
+    echo 'precedence ::ffff:0:0/96 100' >> /etc/gai.conf
```

Comment:
reading man gai.conf it sounds like we need to repeat all the other defaults precedence values as well, and not just change this one